### ISSUE TITLE
SphereProgress broke with latest patch

### ISSUE THREAD

**Issue by randomlypickedname** [fezhub/DSP-Mods#42]
Exception upon opening Dyson Sphere panel:

```
Exception hit 413 times: System.MissingMethodException: Method not found: 'DysonSphere.get_totalConstructedPoint'.


at (wrapper dynamic-method) UIDysonPanel.DMD<UIDysonPanel.._OnUpdate> (UIDysonPanel) <0x0041c>
at ManualBehaviour._Update () <0x0006a>

Target methods matching by name:
virtual System.Void UIDysonPanel::_OnUpdate()

Relevant plugins:
1. org.fezeral.plugins.sphereprogress - thisisbrad-SphereProgress\SphereProgress.dll
	Prefix[index=0]: static System.Void DSP_Mods.SphereProgress.PatchSphereProgress::UIDysonPanel_OnUpdate_Postfix(UIDysonPanel __instance)

==================
```

**Comment by GlitchHound** [comment]
Addressed in PR #43, 🤞 it can be pulled in shortly.

**Comment by fezhub** [comment]
Thanks Glitch, fixed in 1.1 release out now
